projects/ndpi/ndpi-reader-afl++.toml

```diff
@@ -20,9 +20,9 @@ target = "/nDPI/sydr/load_sydr_ndpi_reader @@"
 jobs = 2
 
 [aflplusplus]
-args = "-t 1000 -i /corpus"
```

Comment:
default timeout is 10ms for afl++, so, there should be some

Reply:
resolved